### ISSUE TITLE
Adjust power and Item Quantity in display mode

### ISSUE THREAD

**Issue by DigiBear** [zombieFox/awesomeSheet#160]
I think the user experience could greatly benefit from letting the user adjust their powers quantity from the statistics section and the items quantity from the equipment page from display mode, possibly in the same way we can adjust if a spell is cast and prepared from the display mode. 

I find that generally, while in play I want to be in edit most the least amount of time since it kind of breaks the flow of the game or combat. instead of "_click > adjust_" it becomes "_edit mode > switch to the right tab > find the right entry > adjust > return to display mode_" so making these actions easier to achieve would greatly improve the usage speed.

Also, and this is way more of a "like to have" than a serious improvement. It would be great if consumables can be linked to items in the item list. that way stuff like torches and rations don't need to be adjusted twice in both the consumables list and the item list to keep everything up to date.

**Comment by zombieFox** [comment]
I agree. I've experienced the same when making changes to Consumables and Powers. This is a quality of life improvement I've been thinking of building. I'll add it to the project board. 



Regarding linking Items to Consumables:
I'm sorry to say this is not a simple task. The Clone module as it stands just isn't built to handle an entry represented in the UI in two locations. What might make sense is to decouple the Item and Consumable logic from [Clone] (https://github.com/zombieFox/awesomeSheet/blob/master/src/js/clone.js) and build a new Inventory Management JS module. I'll add this to the project board as an investigator task. If the outcome is too large for now it will be put on the back burner until I can complete other tasks on the board. Thanks again for all the input!
